clickhouse_connect/dbapi/connection.py

```diff
@@ -29,6 +29,10 @@ def __init__(self,
                                     secure=secure,
                                     dsn=dsn,
                                     generic_args=kwargs)
+        try:
```

Comment:
Doesn't seem necessary to catch an exception here.  We have full control over the _add_integration_tag method and it shouldn't throw anything.